sklearn/neighbors/tests/test_dist_metrics.py

```diff
@@ -89,12 +99,15 @@ def check_cdist_bool(metric, D_true):
 
 
 @pytest.mark.parametrize('metric', METRICS_DEFAULT_PARAMS)
-def test_pdist(metric):
+@pytest.mark.parametrize('X1, X2', [(X1, X2), (X1_mmap, X2_mmap)])
+def test_pdist(metric, X1, X2):
     argdict = METRICS_DEFAULT_PARAMS[metric]
     keys = argdict.keys()
     for vals in itertools.product(*argdict.values()):
         kwargs = dict(zip(keys, vals))
-        if metric == "wminkowski":
+        if metric == "mahalanobis":
+            pytest.xfail("cdist with 'mahalanobis' fails on memmap data")
```

Comment:
Same here regarding linking to an issue.

Should the message be `pdist` in this case?

```suggestion
        if metric == "mahalanobis":
            pytest.xfail("pdist with 'mahalanobis' fails on memmap data")
```

Reply:
I previously used `pdist`, but changed to `cdist` as `pdist` is just an interface.

I will revert this.

Reply:
I got confused looking at the way tests are setup: as `pdist` is never called, I inferred that `pdist` was an interface of `cdist`, hence this change), but this is not the case.

I will revert it.